components/ChannelList/ChannelList.jsx

```diff
@@ -96,7 +96,7 @@ export default class ChannelList extends React.Component {
           opacity = 0;
         }
         else if (subIndex !== -1) {
-          icon = require('../../assets/Bell-ON.png');
+          icon = require('../../assets/bell-ON.png');
```

Comment:
`bell-ON.png` doesn't exist, the capitalisation doesn't match the actual asset.

Reply:
I tried to make the filenames consistent, but forgot that windows has case insensitive filesystem. Should be fixed now.